intermediate/indexing/advanced-indexing.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>I would note that there are no shared dimension names between <code>target_lat</code> and <code>target_lon</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Good point! I updated this as following:

In the above example, you can see how the output shape is `time` x `lats` x `lons`. Please note that there are no shared dimensions between the indexers, so the output shape is the union of the dimensions of the indexers.